wire/common.go

```diff
@@ -128,6 +129,19 @@ func ReadElement(r io.Reader, element interface{}) error {
 		}
 		return nil
 
+	case **id.ID:
```

Comment:
Why would we ever pass a double-pointer here?

Reply:
Otherwise we need to copy the whole ID, but I guess it's ok